great_expectations/data_context/data_context/context_factory.py

```diff
@@ -37,11 +37,300 @@
         EphemeralDataContext,
         FileDataContext,
     )
+    from great_expectations.data_context.store import Store
     from great_expectations.data_context.types.base import DataContextConfig
 
 ContextModes: TypeAlias = Literal["file", "cloud", "ephemeral"]
 
 
+class ProjectManager:
```

Comment:
If we've decided on this name, can we change the name of the file?

Reply:
i haven't heard an explicit call one way or another, so this PR is inconsistent with the naming. will update once we have a clear direction.